python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -161,27 +163,16 @@ def _execute_action(self, data):
         """
         if data["name"] == "__clone_pc":
             logger.debug("Clone configuration command received.")
-            try:
-                self._clone_configuration(data)
-            except Exception as e:
-                self.host.reply(
-                    dict(
-                        retcode=constants.COMMAND_FAILED,
-                        err=str(e),
-                        out=str(e),
-                    ),
-                )
-                raise
-            else:
-                logger.debug("Clone configuration successful.")
-                self.host.reply(
-                    dict(
-                        retcode=constants.COMMAND_SUCCEEDED,
-                        err="",
-                        out="",
-                    ),
-                )
-                return
+            self._clone_configuration(data)
```

Comment:
Note: This is just cleanup. The handling of the exception here isn't needed any longer now that exception handling is more robust in this method's caller.